example/server.js

```diff
@@ -16,13 +16,24 @@
 //    NODE_ENV=development node example/server.js
 
 const http = require('http')
+const https = require('https')
 const env = require('./env')
 
-const server = http.createServer((req, res) => {
+var options = {
+  key: fs.readFileSync('test/fixtures/keys/agent2-key.pem'),
+  cert: fs.readFileSync('test/fixtures/keys/agent2-cert.cert')
+};
+
+const serverBody = (req, res) => {
   res.statusCode = 200
   res.setHeader('Content-Type', 'text/plain')
   res.end(env.MESSAGE)
-})
+}
+
+const server = (env.USE_TLS)? http.createServer(serverBody): https.createServer({
+  key: fs.readFileSync(env.TLS_KEY_PATH),
+  cert: fs.readFileSync(env.TLS_CERT_PATH)
+},serverBody);
```

Comment:
Could you roll back the changes to this example file? IMO they make this very simple file harder to follow

Reply:
sure thing!